config/rollup.config.js

```diff
@@ -70,13 +61,9 @@ export default (async () => ({
     }),
     json(),
     postcss({
-      extract: 'dist/style.css',
+      extract: 'style.css',
```

Comment:
updated v of postcss builds to dist anyway... this was building to dist/dist/style.css